README.md

```diff
@@ -20,58 +18,42 @@ which is installed automatically for new Gatsby sites.
 > - Essential Gatsby is not compatible with the Gatsby community plugin
 >   [gatsby-plugin-netlify-cache](https://www.gatsbyjs.com/plugins/gatsby-plugin-netlify-cache/).
 
-## Installation and Configuration
+## Installation
 
-<!-- All sites deployed to Netlify with Gatsby will automatically install this plugin
-for a seamless experience.
+Gatsby sites need two plugins to support all features.
 
-This means that you don't have to do anything — just build and deploy your site
-to Netlify as usual and we'll handle the rest.
+1. The Netlify build plugin, called "Essential Gatsby" or
+   `@netlify/plugin-gatsby`. This is installed automatically for all Gatsby
+   sites deployed to Netlify.
+2. The Gatsby plugin `gatsby-plugin-netlify`, which needs to be manually
+   installed.
 
-You're able to
-[remove the plugin](https://docs.netlify.com/configure-builds/build-plugins/#remove-a-plugin)
-at any time by visiting the **Plugins** tab for your site in the Netlify UI. -->
+### Installing the Netlify build plugin
```

Comment:
[sand] This header uses "Installing" while the following header (line 43) uses "Install". They should use the same verb form. I advocate using "installing" for both, but you could make an argument either way.